src/ontology/mondo-edit.obo

```diff
@@ -306326,10 +306334,12 @@ xref: SCTID:46567003 {source="DOID:2907"}
 xref: UMLS:C0265240 {source="DOID:2907", source="Orphanet:374/e", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:164210", source="NCIT:C84740", source="Orphanet:374"}
 xref: UMLS:C0432130 {source="Orphanet:374/e", source="MONDO:notFoundInDiseaseSubset", source="Orphanet:374"}
 xref: UMLS:C3495417 {source="MONDO:notFoundInDiseaseSubset", source="OMIM:164210"}
+xref: UMLS:CN199493 {source="MONDO:equivalentTo"}
```

Comment:
Is this `equivalentTo` an issue given Line 306334?

Reply:
Good catch, this is a proxy merge. I will fix this.

Reply:
I changed this to MONDO:obsoleteEquivalentObsolete because it is obsolete in UMLS and we obsoleted the class in Mondo (and merged it into this class)